docs/protocol.md

```diff
@@ -1,211 +1,371 @@
+| Check $NR = \tilde{NR}, J_{sig,pk} = \tilde{J}, S = \tilde{S}$                                                           |                        |                                  |
+| Discard $J_{edh,sk}, J_{ekem,sk}, J_{fetch,sk}$                                                                          |                        |                                  |
+| Return $msg \Vert S_{dh,pk} \Vert S_{kem,pk} \Vert S_{fetch,pk}$                                                         |                        |                                  |
+
+### 9. Journalist replies to a source
+
+For some message $msg$ in reply to a source $S$:
+
+| Journalist                                                                                               |                             | Server                           |
+| -------------------------------------------------------------------------------------------------------- | --------------------------- | -------------------------------- |
+| $`m \gets msg \Vert S \Vert J_{sig,pk} \Vert J_{fetch,pk} \Vert J_{dh,pk} \Vert \sigma^{NR} \Vert NR`$   |                             |                                  |
+| $`((c_1, c_2), C'') \gets^{\$} \text{AuthEnc}(J_{dh,sk}, (S, S_{kem,pk}), m, \varepsilon, \varepsilon)`$ |                             |                                  |
+| $`C' \gets ^{\$} \text{Enc}(S_{pke,pk}, J_{dh,pk} \Vert c_1 \Vert c_2)`$                                 |                             |                                  |
+| $`C \gets C' \Vert C''`$                                                                                 |                             |                                  |
+| $`x \gets^{\$} \mathbb Z_q`$                                                                             |                             |                                  |
+| $`Z \gets \text{DH}(S_{fetch,pk}, x)`$                                                                   |                             |                                  |
+| $`X \gets \text{DH}(g, x)`$                                                                              |                             |                                  |
+|                                                                                                          | $`\longrightarrow C, Z, X`$ |                                  |
+|                                                                                                          |                             | $`id \gets^{\$} \text{Rand}()`$  |
+|                                                                                                          |                             | $`messages[id] \gets (C, Z, X)`$ |
+
+> [!NOTE]
+> As a mitigation against traffic analysis, in addition to sending the reply
```

Comment:
I'm not sure this mitigates a lot in the end, I think at this point it's rather not to implement groups, just because I'd be cautious with the claim.

Reply:
Good point.  Corrected in a10465ca6129c693c15497af753d883e2c562b12.